### DISCUSSION TITLE
Curso: Fundamentos Blazor co, .net 8 / aula: Novo template do blazor WebAssembly

### DISCUSSION THREAD

**Discussion by dhionys-soares** [balta-io/forum#613] [Tira Dúvidas] [answered]
Dúvida: Como a página Counter(que é uma página WebAssembly) está dentro do projeto Client e é referenciada no program no AddAdditionalAssemblies(), então, se eu quiser criar outra página WebAssembly, eu teria que criar no projeto Client também?

**Comment by andrebaltieri** [comment, marked as answer]
@dhionyssoares bom dia, como vai?

Na verdade não é uma página... o WebAssembly gera um binário com aplicação completa e pode conter várias páginas.
Não é necessário alterar o projeto.

Recomendo fazer os cursos da carreira .NET (Todos e na ordem):
👉 https://balta.io/carreiras
